src/node.cc

```diff
@@ -3398,6 +3402,8 @@ static void ParseArgs(int* argc,
     } else if (strcmp(arg, "--v8-options") == 0) {
       new_v8_argv[new_v8_argc] = "--help";
       new_v8_argc += 1;
+    } else if (strncmp(arg, "--v8-pool-size=", 15) == 0) {
+      v8_thread_pool_size = arg + 15;
```

Comment:
Couldn't you set the pool size to the default of 4 when it's declared and then update the value here? I **think** that is what @bnoordhuis is getting at.